cluster/src/test/scala/org/apache/spark/sql/store/BugTest.scala

```diff
@@ -475,5 +475,68 @@ class BugTest extends SnappyFunSuite with BeforeAndAfterAll {
 
   }
 
+  test("Bug SNAP-2887") {
+    snc
+    var serverHostPort2 = TestUtil.startNetServer()
+    var conn = DriverManager.getConnection(s"jdbc:snappydata://$serverHostPort2")
+    var stmt = conn.createStatement()
+    val snappy = snc.snappySession
+
```

Comment:
Why so many spaces throughout the test file.

Reply:
removed those